### ISSUE TITLE
Adds the possibility to reply or forward from the EnvelopeList component

### ISSUE THREAD

**Issue by StCyr** [nextcloud/mail#5225]
### Feature Request

Before some recent changes it was possible to reply to a message or forward it directly from the EnvelopeList.

I would like to get that possibility back.

#### Summary

- [ ] When the user clicks the reply or forward action, Mail shall reply/forward the latest (more recent) message in the thread.
- [ ] Additional question: Shouldn't the "download thread data for debugging" action be moved in the EnvelopList's action menu (it's now added to every ThreadEnvelope)?
- [ ] The action menu from the EnvelopeList component and the one from the threadEnvelope component should be streamlined to show as much the same actions as possible, and in the same order
- [ ] The action menu that appears when the user selects several messages/thread should probably also get the "mark important", "mark as spam" and "edit tags" actions
- [ ] Is it normal that the action is called "edit tags" in EnvelopeList but "add tags" in ThreadEnvelope?

**Comment by ChristophWurst** [comment]
Sounds related to the new list item component change. we had to break your deduplication into two separately maintained lists of actions again. Moreover there were some changes in regards to messages vs threads. Most action in the envelope list now operate on a thread level instead of message level.

cc @GretaD @kesselb

**Comment by GretaD** [comment]
Yes, this is because we removed MenuEnvelope component from Envelope when we start using ListItem and some actions were left out because looks like i applied the old menu.

**Comment by StCyr** [comment]
> Yes, this is because we removed MenuEnvelope component from Envelope when we start using ListItem because we couldnt use MenuEnvelope inside ListItem and some actions were left out because looks like i applied the old menu.

Ah ok. I understand. Thanks for letting me know the reason behind this change 👍 

I'll see what I can do to improve things a bit.

**Comment by ChristophWurst** [comment]
@jancborchardt @nimishavijay could you please tell us what actions we should expose from the message list?

![Bildschirmfoto vom 2022-05-17 08-44-58](https://user-images.githubusercontent.com/1374172/168746434-0976101e-49d3-479b-b964-3eb6cb35b5f2.png)

^ v1.13.0 Beta 1 has these operations.

We could add *Reply*/*Reply all*/*Reply to sender only* and *Forward* as secondary actions and move *Move thread* into *More actions*.
